protos/dcs/coalition/v0/coalition.proto

```diff
@@ -121,6 +131,58 @@ message AddGroupResponse {
   dcs.common.v0.Group group = 1;
 }
 
+message AddStaticObjectRequest {
+  // the name of the static; must be unique or would destroy previous object
+  string name = 1;
+  // country the unit belongs to
+  dcs.common.v0.Country country = 2;
+  // type of the static object (e.g. "Farm A", "AS32-31A")
+  string type = 3;
+  // string name of the livery for the aircraft
+  string livery = 4;
+  // boolean for whether or not the object will appear as a wreck
+  bool dead = 5;
+  // number value for the "score" of the object when it is killed
+  optional uint32 score_rate = 6;
+
+  double heading = 7;
+  dcs.common.v0.Position position = 8;
+  // cargo mass in kilograms
+  uint32 cargo_mass = 9;
+}
+
+message AddStaticObjectResponse {
+  string name = 1;
+}
+
+message AddLinkedStaticRequest {
+  // the name of the static; must be unique or would destroy previous object
+  string name = 1;
+  // country the unit belongs to
+  dcs.common.v0.Country country = 2;
+  // type of the static object (e.g. "Farm A", "AS32-31A")
+  string type = 3;
+  // string name of the livery for the aircraft
+  string livery = 4;
+  // boolean for whether or not the object will appear as a wreck
+  bool dead = 5;
+  // number value for the "score" of the object when it is killed
+  optional uint32 score_rate = 6;
+  // the name of the unit to offset from
+  string unit = 7;
+  // the angle to relative to the linked unit
+  double angle = 8;
```

Comment:
Or if we consider renaming it, how about: `orientation`?

Reply:
Other ideas/suggestions:

`direction`
`alignment_angle`

Reply:
I guess the name is less important than we make sure the documentation on the field explicitly calls out what the values mean because I am still not 100% sure. For example I am assuming clockwise heading offset but can we do negatives? is -90 and 270 the same thing?

Reply:
I tested with negative values. It is anti-clockwise.